src/rsz/test/repair_setup6.ok

```diff
@@ -17,8 +17,9 @@ delta HPWL                    0 %
 worst slack -0.39
 [INFO RSZ-0094] Found 4 endpoints with setup violations.
 [INFO RSZ-0099] Repairing 4 out of 4 (100.00%) violating endpoints...
-[INFO RSZ-0059] Removed 5 buffers.
-[INFO RSZ-0041] Resized 1 instances.
+[INFO RSZ-0059] Removed 3 buffers.
+[INFO RSZ-0040] Inserted 3 buffers.
+[INFO RSZ-0041] Resized 2 instances.
 Repair timing output passed/skipped equivalence test
 worst slack 0.01
-worst slack 0.01
+worst slack -0.10
```

Comment:
Any idea why this degraded slightly?

Reply:
Resizer is removing fewer buffers with my update. I guess preserving the net topologies helps avoid extra buffer removals, but the QoR is affected.